dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/TaskInstance.java

```diff
@@ -187,6 +189,19 @@ public class TaskInstance implements Serializable {
      */
     private Flag flag;
 
+    /**
+     * task is cache: yes/no
+     */
+    private IsCache isCache;
+
+    /**
+     * task is cache: yes/no
+     */
+    @TableField(updateStrategy = FieldStrategy.IGNORED)
```

Comment:
Why add the ignored update strategy for it?

Reply:
The default value of cache_key is `null`, if we remove the cache_key, we have to set it to `null`. So we have to ignore the update strategy, if not, the `null` will not be updated, that means cache data will not be removed.